custom_components/hacs/base.py

```diff
@@ -674,12 +674,12 @@ async def startup_tasks(self, _=None) -> None:
             async_track_time_interval(self.hass, self.async_check_rate_limit, timedelta(minutes=5))
         )
         self.recurring_tasks.append(
-            async_track_time_interval(self.hass, self.async_process_queue, timedelta(minutes=10))
+            async_track_time_interval(self.hass, self.async_process_queue, timedelta(minutes=30))
         )
 
         self.recurring_tasks.append(
             async_track_time_interval(
-                self.hass, self.async_handle_critical_repositories, timedelta(hours=6)
+                self.hass, self.async_handle_critical_repositories, timedelta(hours=24)
```

Comment:
This one does not match the description.
This handler will (now) fetch this file: https://github.com/hacs/default/blob/master/critical>. However, starting from the next version, this will not hit GitHub at all, as a different API (https://data-v2.hacs.xyz/critical/data.json>) will serve this up.